py-wacz/tests/fixtures/pages.jsonl

```diff
@@ -0,0 +1,3 @@
+{"format": "json-pages-1.0", "id": "pages", "title": "All Pages"}
```

Comment:
should this file be added, or accidental add?

Reply:
yeah this was an accident I was using it locally